site/content/api/Telemetry-API.md

```diff
@@ -65,6 +65,9 @@ The following table provides an overview of the properties a client needs to set
 | *content-type* | yes       | *properties*             | *symbol*  | SHOULD be set to *application/octet-stream* if the message payload is to be considered *opaque* binary data. In most cases though, the client should be able to set a more specific content type indicating the type and characteristics of the data contained in the payload, e.g. `text/plain; charset="utf-8"` for a text message or `application/json` etc. |
 | *device_id*    | yes       | *application-properties* | *string*  | MUST contain the ID of the device the data in the payload has been reported by. |
 | *reg_assertion*| yes       | *application-properties* | *string*  | A [JSON Web Token](https://jwt.io/introduction/) issued by the [Device Registration service]({{< relref "api/Device-Registration-API.md#assert-device-registration" >}}) asserting the device's registration status. |
+| *connection_timestamp* | no       | *application-properties* | *string*  | The point in time that defines the beginning of a time interval for which a device may receive upstream data (e.g. a command). The value MUST be an ISO 8601 compliant combined date and time representation in extended format. **NB:** due to network latencies this timestamp will typically be some time in the past already when the event is received by the application. |
```

Comment:
why can't we simply use the standard *creation-time* property?

BTW these properties also need to be supported by the Event API.

Reply:
Good point, we can indeed use the creation-time property. Will change that.

The Event API should support the `stay-connected` property as well, agree on that.